### DISCUSSION TITLE
Github 2FA and lost phone

### DISCUSSION THREAD

**Discussion by mateusvieites** [community/community#135246] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I lost my old phone and had to buy a new one. When I installed GitHub, I was surprised to find that neither Google Authenticator nor the GitHub mobile app was available. Now, when I try to log in, it asks for confirmation from the old phone.
Don’t have access to either Google Authenticator or the GitHub mobile app. How can I resolve this? I've sent a message to support but haven’t received a response for a month. 
I live in Brazil, so don't have access to SMS recovery

**Comment by dane** [comment]
Google Authenticator and GitHub both offer a recovery mechanism for situations just like this. Here's some information on [Google's recovery process](https://support.google.com/accounts/thread/217906635/how-do-i-recover-authenticator-codes?hl=en) and here is information on [GitHub's process](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials).

**Reply by mateusvieites** [reply]
Yeah, but it looks like Google Auth isn't syncing with GitHub, and I've already tried all the steps GitHub suggests. I ended up creating a support ticket, but it's still unresolved.
